devel/reuse/Portfile

```diff
@@ -32,5 +32,6 @@ depends_lib-append  port:py${python.version}-binaryornot \
                     port:py${python.version}-boolean.py \
                     port:py${python.version}-jinja2 \
                     port:py${python.version}-license-expression \
-                    port:py${python.version}-python-debian \
```

Comment:
the "python.versions" above doesn't do anything here since the port is not named "py-<portname>". Instead it should set "python.default_version 312" (Python 3.12 is MacPorts' defuslt and it should be used if possible).

Reply:
Thank you for review.

I fixed the commit to set "python.default_version 312".